inscriptions/ISic0728.xml

```diff
@@ -189,7 +189,7 @@
             </div>
             
             <div type="apparatus">
-                <p>Text based on Manganaro (1989), p. 181-2 fig. 66a. Note that it is not obvious that these two fragments come from the same text, but they are always published together.</p>
+                <p>Text based on Manganaro 1989. Note that it is not obvious that these two fragments come from the same text, but they are always published together.</p>
```

Comment:
As previous for encoding apparatus. I think that the longer comment should be in the commentary and not in the apparatus.

Reply:
Fixed